ProfileManager.js

```diff
@@ -382,15 +381,13 @@ export default class ProfileManager {
       return;
     }
 
-    // cache account capability agent
-    const {secret} = (authentication || {});
-    this.capabilityAgent = await (secret ?
-      _getCapabilityAgentFromSecret({secret, accountId}) :
-      CapabilityAgent.fromCache({handle: accountId}));
```

Comment:
Hmm, we still need to pull from cache here. We should always pull from cache if available, IMO. We don't have to worry about the secret changing any more (and deciding not to pull from cache when that happens) -- as that was related to it being based on a password.